src/modules/70-pipeline/components/PipelineSteps/Steps/ServiceNowUpdate/types.ts

```diff
@@ -47,6 +55,10 @@ export interface ServiceNowUpdateData extends StepElementConfig {
   }
 }
 
+export enum TaskTypes {
```

Comment:
enum should be singular
```suggestion
export enum TaskType {
```